src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -193,10 +225,13 @@ public MatrixFactorizationTrainer(IHostEnvironment env, Arguments args) : base(e
             Host.CheckUserArg(args.Lambda > 0, nameof(args.Lambda), posError);
             Host.CheckUserArg(args.Eta > 0, nameof(args.Eta), posError);
```

Comment:
>CheckUserArg [](start = 17, length = 12)

Do you wanna add Checks for your new added parameters here?

Reply:
You added check for Alpha, what about C? 
Is it also only positive value?

Can C and Alpha be greater than 1?

---
In reply to: [235105129](https://github.com/dotnet/machinelearning/pull/1664#discussion_r235105129) [](ancestors = 235105129)